traffic_router/core/src/test/conf/log4j2.xml

```diff
@@ -18,15 +18,19 @@
         <Console name="stdout" target="SYSTEM_OUT">
             <PatternLayout pattern="%m%n" />
         </Console>
-        <RollingFile name="traffic_router_access" fileName="/opt/traffic_router/var/log/access.log" >
+        <RollingFile name="traffic_router_access"
+                     fileName="/opt/traffic_router/var/log/access.log"
+                     filePattern="/opt/traffic_router/var/log/access-%d{yyyyMMdd}.log" >
             <PatternLayout pattern="%m%n" />
             <Policies>
                 <SizeBasedTriggeringPolicy size="200MB" />
             </Policies>
             <DefaultRolloverStrategy max="10"/>
             <ThresholdFilter level="INFO"/>
         </RollingFile>
-        <RollingFile name="traffic_router" fileName="${deploy.dir}/var/log/traffic_router.log" >
+        <RollingFile name="traffic_router"
+                     fileName="${deploy.dir}/var/log/traffic_router.log"
+                     filePattern="${deploy.dir}/var/log/traffic_router-%d{yyyyMMdd}.log" >
```

Comment:
There should be a dash separating the year, month and day), like ISO 8601: `yyyy-MM-dd`